etools-searchable-multiselection-menu.html

```diff
@@ -551,6 +567,22 @@
           if (emptyValue) {
             filtered.unshift({value: null, label: '-- None --', style: 'emptyValue'});
           }
+          if (this.twoLines) {
+            var newOptions = filtered.map(function(item) {
+              var split = item.label.split('|');
+              return {
+                value: item.value,
+                label: split[0],
+                display: {
+                  primary: split[0],
+                  secondary: split[1]
+                },
+                style: 'two-lines'
+              }
+            });
+            this.set('twoLinesOptions', newOptions);
```

Comment:
At first look it doesn't seem like you need the twoLinesOptions property , you can just store the values in 'options' property